website/src/theme/CodeBlock/index.js

```diff
@@ -88,54 +57,138 @@ export default ({
     };
   }, [button.current, target.current]);
 
+  useEffect(() => {
+    async function getDependencies() {
+      const { TezosToolkit, MichelsonMap, compose, DEFAULT_FEE, RpcReadAdapter } = await import('@taquito/taquito');
+      const { verifySignature } = await import('@taquito/utils');
+      const {
+        validateAddress,
+        validateChain,
+        validateKeyHash,
+        validateContractAddress,
+        validatePublicKey,
+        validateSignature,
+        validateBlock,
+        validateProtocol,
+        validateOperation,
+        b58cencode,
+        prefix,
+        Prefix
+      } = await import('@taquito/utils');
+      const { BeaconWallet } = await import('@taquito/beacon-wallet');
```

Comment:
for dynamic imports, wouldn't we only need to do this for the `beacon-wallet` package? does it need to dynamically import all packages?

Reply:
I was thinking that any of the packages can later change a dependency and break the website. So maybe loading all of them is not a terrible idea after all.

Reply:
@dsawali 
There is something we can discuss. Here, all `async` library loads are done one after the other. An alternative would be to run all the `import` functions, get the `Promise` objects and `awaitAll` them.

This makes the code a bit harder to follow, but would possibly get a performance benefit. In production code it could be useful, but for the website, I can't be sure.

What do you suggest?

Reply:
If we can leverage that for performance, that probably would be nice. But I'm not sure if that would reflect in our website. Docusaurus handles most of the page rendering. I am not opposed to the change, however.